templates/cluster/vsphere-standalone-cp/templates/cluster.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
 metadata:
   name: {{ include "cluster.name" . }}
+  labels:
+    {{- with .Values.clusterLabels }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
nit: i think it'd look a little bit more beautiful with just `indent` instead of the `nindent` (but please verify it). applicable for the rest of the templates' clusters 

I also think this could be simplified to something like (but should be verified as well)
```yaml
  {{- if .Values.clusterLabels }}
  labels: {{- toYaml .Values.clusterLabels | nindent 4 }} # here nindent on purpose
  {{- end }}
```